go.mod

```diff
@@ -18,11 +18,6 @@ require (
 	gopkg.in/redis.v4 v4.2.4
 )
 
-require (
-	golang.org/x/mod v0.6.0-dev.0.20220419223038-86c51ed26bb4 // indirect
-	golang.org/x/tools v0.1.12 // indirect
-)
-
```

Comment:
We shouldn't have two "indirect" section in the go.mod. For an unknown reason, sometimes `go get` does that :/